src/components/Form/Form.md

```diff
@@ -39,24 +38,31 @@ import { TitleLarge, TextInput, Spacer, Button, InfoMessage, ZipInput, ZipInputV
     },
     onSubmit: async (formData) => {
       await new Promise((resolve) => setTimeout(resolve, 500))
-      if (!!(Math.floor(Math.random() * 10) % 2)) {
-        throw new Error("Oh no, the api is broken (try again, it's random)")
+      if (count++ % 2 === 0) {
```

Comment:
Multiple requests to make this  deterministic (sorry @kenning  😛 cc / @anbright @ekeric13 I believe y'all provided this feedback)